soundata/datasets/mavd.py

```diff
@@ -0,0 +1,303 @@
+
+
+    Annotation Files Included
+    -------------------------
+    The annotations list the sound events that occur in every recording and are provided as a text file for each recording (.csv file). 
+    The annotations are "strong", meaning for every sound event the annotations include the start time, end time, and label of the sound event. 
+
+    The sound event annotations follow an ontology for traffic sounds that is the combination of a set of two taxonomies:
+        * vehicle types (e.g. car, bus) and vehicle components (e.g. engine, brakes),
+        * and a set of actions related to them (e.g. idling, accelerating).
+     Since the taxonomies follow a hierarchy it can be used with different levels of detail. 
+
+    * Further details about the ontology for traffic noise description can be found in Section 2 of the aforementioned paper.
+    * Further details about the annotation process using the ELAN software can be found in Section 3.2 of the aforementioned paper.
+
+
+    Please Acknowledge MAVD-traffic in Academic Research
+    ----------------------------------------------------
+
+    When MAVD-traffic is used for academic research, we would highly appreciate it if scientific publications of works 
+    partly based on the MAVD-traffic dataset cite the following publication:
+
+    .. code-block:: latex
+        P. Zinemanas, P. Cancela, and M. Rocamora. "MAVD: A dataset for sound event detection in urban environments", 
+        In Detection and Classification of Acoustic Scenes and Events (DCASE), New York, NY, USA, Oct. 2019.
+
+
+    Conditions of Use
+    -----------------
+
+    Dataset created by P. Zinemanas, P. Cancela, and M. Rocamora.
+    
+    The MAVD-traffic dataset is offered free of charge under the terms of the Creative Commons
+    Attribution 4.0 International License (CC BY 4.0): http://creativecommons.org/licenses/by/4.0/
+    
+    The dataset and its contents are made available on an "as is" basis and without warranties of any kind, including 
+    without limitation satisfactory quality and conformity, merchantability, fitness for a particular purpose, accuracy or 
+    completeness, or absence of errors. Subject to any liability that may not be excluded or limited by law, 
+    Universidad de la República is not liable for, and expressly excludes, all liability for loss or damage however and 
+    whenever caused to anyone by any use of the MAVD-traffic dataset or any part of it.
+
+
+    Feedback
+    --------
+
+    Please help us improve MAVD-traffic by sending your feedback to: rocamora@fing.edu.uy
+    In case of a problem report please include as many details as possible.
+
+"""
+
+import os
+from typing import BinaryIO, Optional, TextIO, Tuple
+
+import librosa
+import numpy as np
+import csv
+import jams
+import glob
+
+from soundata import download_utils
+from soundata import jams_utils
+from soundata import core
+from soundata import annotations
+from soundata import io
```

Comment:
All imports from same library in one line

Reply:
OK. Done.